.github/workflows/integration-tests.yml

```
@@ -0,0 +1,26 @@
+name: "Run integration tests for read-db"
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK 'zulu' '11'
+        uses: actions/setup-java@v3
```

note: v3 uses the old node version
```suggestion
        uses: actions/setup-java@v4
```